content/_data/changelogs/lts.yml

```diff
@@ -3913,6 +3913,590 @@
+    authors:
+    - daniel-beck
+    message: |-
+      Add the ability to filter out environment variables for Shell and Windows batch build steps.
+  - type: rfe
+    category: rfe
+    pull: 4841
+    issue: 61806
+    authors:
+    - jglick
+    message: |-
+      Show hints about the current time on the server’s zone when configuring your <b>User Defined Time Zone</b>.
+  - type: bug
+    category: bug
+    pull: 4833
+    issue: 62723
+    authors:
+    - jglick
+    - Zastai
+    message: |-
+      Fix <code>IllegalArgumentException: Method not found</code> error caused by misbehaviour in <code>Util.isOverridden()</code> (regression in 2.241).
+  - type: bug
+    category: bug
+    pull: 4830
+    authors:
+    - jglick
+    message: |-
+      Avoid a stack trace when an older version of glibc was used by an agent.
+  - type: bug
+    category: bug
+    pull: 4839
+    issue: 63014
+    authors:
+    - jglick
+    message: |-
+      Remove the fallback Jenkins URL from the JNLP launch file so that WebSocket agents can be connected over Java Web Start.
+
+  # Fingerprint improvements
+  - type: rfe
+    category: rfe
+    pull: 4834
+    authors:
+    - stellargo
+    message: |-
+      Allow fingerprint storage engine to be selected from the configuration page.
+    references:
+      - issue: 63022
+      - url: https://github.com/jenkinsci/jep/tree/master/jep/226
+        title: JEP-226
+  - type: rfe
+    category: developer
+    pull: 4731
+    authors:
+    - stellargo
+    references:
+      - issue: 62345
+      - pull: 4731
+      - url: https://github.com/jenkinsci/jep/tree/master/jep/226
+        title: JEP-226
+      - url: https://github.com/jenkinsci/redis-fingerprint-storage-plugin
+        title: Reference Implementation
+    message: |-
+      Developer: Add experimental External Fingerprint Storage API.
+  - type: rfe
+    category: developer
+    pull: 4731
+    authors:
+    - stellargo
+    references:
+      - issue: 62345
+      - pull: 4731
+      - url: https://javadoc.jenkins.io/hudson/model/Fingerprint.html
+        title: Fingerprint API Javadoc
+    message: >
+      Developer: Introduce new Fingerprint API methods: <code>Fingerprint#getPersistedFacets()</code>, <code>Fingerprint#delete(String)</code> and file-independent CRUD methods.
+      Deprecate <code>Fingerprint#save(File)</code>.
+  - type: rfe
+    category: developer
+    pull: 4834
+    issue: 63022
+    authors:
+    - stellargo
+    message: |-
+      Developer: Require FingerprintStorage descriptors so they can be configured from the configuration page.
+  - type: rfe
+    category: rfe
+    pull: 4842
+    issue: 62747
+    authors:
+    - timja
+    message: |-
+      Developer: Add <code>alert-success</code> banner.
+  - type: rfe
+    category: developer
+    pull: 4683
+    issue: 62014
+    authors:
+    - daniel-beck
+    message: |-
+      Developer: Add new extension points to define build step environment filters (currently in beta).
```

Comment:
Would not consider this an important enough change to make this changelog; Javadoc and weekly changelog are generally better sources for developer enhancements unless we need people to be aware.